src/main/java/seedu/medmoriser/logic/commands/ClearCommand.java

```diff
@@ -19,6 +19,9 @@
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
+        for (int i = 0; i < model.getFilteredQAndAList().size(); i++) {
+            model.getFilteredQAndAList().get(i).setNotQuiz();
+        }
```

Comment:
@ChengJiyuqing sorry but I'm not sure what's this change to all the commands, could you help me to understand what's this for?

Reply:
this is deleted now! It's to reset the cells of displayed qAndA to normal cell